src/tools/common/sss_tools.c

```diff
@@ -300,6 +263,49 @@ void sss_tool_usage(const char *tool_name, struct sss_route_cmd *commands)
     sss_tool_print_common_opts(min_len);
 }
 
+static int tool_cmd_init(struct sss_tool_ctx *tool_ctx,
+                         struct sss_route_cmd *command)
+{
+    int ret;
+
+    if (command->flags & SSS_TOOL_CMD_FLAG_NO_CONFDB) {
+        /* This tool does not need to connect to confdb.
```

Comment:
Sorry, I am out of ideas here. What name do you propose? Or will it be enough if I just add a comment to the flag that it will not initialize the domain context?